src/Functional.CQS.AOP.CommonTestInfrastructure/DummyObjects/DummyAsyncCommandError.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Functional.CQS.AOP.CommonTestInfrastructure.DummyObjects
+{
+	/// <summary>
+	/// 
```

Comment:
If this class has a blank `<summary>`, it may be worthwhile to just delete the comment.

Reply:
Same comment goes for other classes, such as `DummyAsyncCommandThatFails`, `DummyAsyncCommandThatSucceeds`, and `DummyAsyncQueryReturnsNull`.

Reply:
The assembly is configured to require documentation for public objects, so I'll just add information when I do some cleanup